Formula/abcl.rb

```diff
@@ -3,6 +3,7 @@ class Abcl < Formula
   homepage "https://abcl.org/"
   url "https://abcl.org/releases/1.7.1/abcl-src-1.7.1.tar.gz"
   sha256 "d51014b2be6ecb5bcaaacda0adf4607a995dd4b6e9e509c8a1f5a998b7649227"
+  license "GPL-2.0-with-classpath-exception"
```

Comment:
This one is on the list of [Deprecated License Identifiers](https://spdx.org/licenses/GPL-2.0-with-classpath-exception.html).